rules/windows/builtin/security/win_security_windows_defender_exclusions_write_access.yml

```diff
@@ -0,0 +1,35 @@
+title: Windows Defender Exclusion Reigstry Key - Write Access Requested
+id: e9c8808f-4cfb-4ba9-97d4-e5f3beaa244d
+related:
+    - id: 46a68649-f218-4f86-aea1-16a759d81820
+      type: derived
+    - id: a33f8808-2812-4373-ae95-8cfb82134978
+      type: derived
+status: test
+description: |
+    Detects write access requests to the Windows Defender exclsuions registry keys. This could be an indication of an attacker trying to request a handle or access the object to write new exclusions in order to bypass security.
```

Comment:
```suggestion
    Detects write access requests to the Windows Defender exclusions registry keys. This could be an indication of an attacker trying to request a handle or access the object to write new exclusions in order to bypass security.
```